index.rst

```diff
@@ -6,24 +6,233 @@
+This increases the risk of a dangerous unintended motion and produces unnecessary image degrading heat.
+The torque hysteresis will also likely produce difficulties in pointing and tracking.
+
+1. It should be verified that the torque reversal near horizon pointing is produced by the premature truncation of the elevation magnets. If corrective action is shown to be impractical, the software interlocks should be modified to allow operation of the balancing units without the elevation pins inserted horizon pointing, and a new procedure produced for balancing the elevation assembly with the equipment available.
+
+2. The cause of the torque jump near zenith pointing should be determined and eliminated if possible. If not the software interlocks should be modified to allow operation of the balancing units without the elevation pins inserted zenith pointing.
+
+3. The cause of the torque hysteresis should be determined and eliminated. As a result of the EUI it is possible that magnets 31 and 32 are operating incorrectly or reporting their torques (current) incorrectly. If this deficiency is not related to the torque hysteresis it should be corrected regardless. The +x and –x should be labelled correctly. If the large torque hysteresis is produced by another phenomenon this still needs to be determined and rectified. The telescope cannot be properly operated under these conditions.
+
+
+
+.. _appendix-possible-causes:
+
+Appendix: other considered causes of the torque behavior
+=========================================================
+
+Elevation breaks
+----------------
+
+We checked for possible contact of the elevation brakes and the TMA during slews causing a dragging effect.
+This was done in a few different ways.
+First, by looking at the wear patterns between the break pads and the elevation axis at different elevations, there seemed to be some variance in the wear but nothing obvious (see images below or `SUMMIT-7581`).
+Then, we took infrared images of the TMA during slews no hot points were detected.
+Finally, we measured the distance between the break pads and the TMA at multiple elevations while the breaks were released finding at no points did the breaks make contact with the axis during a slew.
+
+.. _SUMMIT-7581: https://jira.lsstcorp.org/browse/SUMMIT-7581
+
+.. image:: _static/tma_brake_plate1.png
+   :width: 32 %
+.. image:: _static/tma_brake_plate2.jpeg
+   :width: 32 %
+.. image:: _static/tma_brake_plate_with_metal_powder.jpg
+   :width: 32 %
+
+A few images of the TMA break plates showing wear, and metal powder.
+
+Elevation Axis Hard Stops
+-------------------------
+We also slewed the telescope to elevations of 7,3 and 0 deg and visually checked whether the hard stops were engaged in a way that could possibly explain the behavior at 3.5 deg.
+It was found this was not possible, and **notably** it looked like the hard stops would engage at a negative **after** the TMA would have encountered portions of the dome floor.
```

Comment:
Isn't this serious? It should not be reported as part of the "open issues"? I know this is a question more for Doug than anyone else, but wanted to raise this question in case you haven't thought about it. Well, this will not affect telescope operation at all, so maybe that is the reason it was not included as an open issue. 

In any case, it means that in the event of an extreme event, we will not have a hard stop to prevent the telescope from touching the floor.

Reply:
Ah yeah, thanks for the reminder, ill follow this up. But for now I will leave the text as is.